src/robusta/integrations/kubernetes/custom_crds.py

```diff
@@ -0,0 +1,130 @@
+from typing import ClassVar
+from pydantic.dataclasses import dataclass
+from hikaru.model.rel_1_26 import HikaruDocumentBase, ObjectMeta, LabelSelector
+import datetime
+import pytz
+from kubernetes import client
+from hikaru.crd import HikaruCRDDocumentMixin
+
+
+@dataclass
+class CRDBase:
+    plural: ClassVar[str] = ""
+    group: ClassVar[str] = ""
+    version: ClassVar[str] = ""
+    total_pods_path: ClassVar[str] = ""
+    ready_pods_path: ClassVar[str] = ""
+
+    metadata: ObjectMeta
+    spec: dict
+    status: dict
+    apiVersion: str
+    kind: str
+
+    @classmethod
+    def readNamespaced(cls, name: str, namespace: str):
```

Comment:
This assumes all CRDs are namespaced resource - but I'm not sure that's the case
I don't think we should change it though